### PR TITLE
Minor refactoring and typos fixing

### PR DESCRIPTION
**Pull request by k1rill-fedoseev**

### CODE REVIEW

File: contracts/upgradeable_contracts/components/common/TokensRelayer.sol [hunk 1]
```diff
@@ -31,8 +31,6 @@ abstract contract TokensRelayer is BasicAMBMediator, ReentrancyGuard {
                 assembly {
                     receiver := calldataload(120)
                 }
-                require(receiver != address(0));
-                require(receiver != mediatorContractOnOtherSide());
```
Comment (akolotov) [inline contracts/upgradeable_contracts/components/common/TokensRelayer.sol:35]: Why did you remove this?

Comment (k1rill-fedoseev) [reply contracts/upgradeable_contracts/components/common/TokensRelayer.sol:35]: I have found a duplicated check in the `bridgeSpecificActionsOnTokenTransfer()` in both HomeOmnibridge and ForeignOmnibridge contracts.

Comment (akolotov) [reply contracts/upgradeable_contracts/components/common/TokensRelayer.sol:35]: ok. thanks

**Review by akolotov — APPROVED** [review]
